packages/grid_client/src/high_level/base.ts

```diff
@@ -86,17 +101,24 @@ class HighLevelBase {
     remainingWorkloads: Workload[],
     deletedMachineWorkloads: Workload[],
     node_id: number,
-  ): Promise<[TwinDeployment[], Workload[], number[], string[], Network | null]> {
+  ): Promise<[TwinDeployment[], Workload[], number[], string[], Network | ZNetworkLight | null]> {
     const twinDeployments: TwinDeployment[] = [];
     const deletedNodes: number[] = [];
     const deletedIps: string[] = [];
     const deploymentFactory = new DeploymentFactory(this.config);
-    let network: Network | null = null;
+    let network: Network | ZNetworkLight | null = null;
+    let contract_id;
+
+    let numberOfIps;
```

Comment:
is there any reason for deaclear the variable here ? i see we are only assigning it once, then check on its value